pkg/api/platformapi/allocatorapi/vacate_full_test.go

```diff
@@ -607,9 +609,115 @@ func TestVacate(t *testing.T) {
 				"\x1b[92;mDeployment [DISCOVERED_DEPLOYMENT_ID] - [Kibana][4ee11eb40eda22cac0cce259625c6734]: finished running all the plan steps\x1b[0m (Total plan duration )",
 			),
 			err: `vacate error: 2 errors occurred:
-	* resource id [3ee11eb40eda22cac0cce259625c6734][elasticsearch] failed vacating, reason: code: a code, message: a message
-	* deployment [DISCOVERED_DEPLOYMENT_ID] - [elasticsearch][5ee11eb40eda22cac0cce259625c6734]: caught error: "Unexpected error during step: [perform-snapshot]: [no.found.constructor.models.TimeoutException: Timeout]"
+	* allocator allocatorID-1: resource id [3ee11eb40eda22cac0cce259625c6734][elasticsearch]: failed vacating: a message (a code)
```

Comment:
Yes it definitely breaks our existing text parsing https://github.com/elastic/soteria/blob/master/app/allocator/vacate.go#L213

Reply:
Why do you think this breaks it?  We actually don't care about the prefix as long as `resource id` or `deployment` is included.  We then proceed to do some string splitting around `[` & `]` depending on which it is.